packages/components-js/tests/vrt/playwright/specs/custom/form-fields.vrt.ts

```diff
@@ -38,6 +41,7 @@ const scenario = async (page: Page, theme: Theme): Promise<void> => {
       .map(([tag, child]) => {
         const childDisabled = child.replace(/((?: \/)?>)/, ' disabled$1');
         const childReadonly = child.replace(/((?: \/)?>)/, ' readonly$1');
+        const isNotWrapperComponent = tag === 'p-pin-code' || tag === 'p-multi-select' ? ' disabled="true"' : '';
```

Comment:
Variable name indicates it is a boolean, however it conditionally contains a disabled attribute.